x/superfluid/keeper/grpc_query.go

```diff
@@ -430,7 +435,7 @@ func (q Querier) SuperfluidDelegationsByValidatorDenom(goCtx context.Context, re
 		lockedCoins := sdk.NewCoin(req.Denom, lock.GetCoins().AmountOf(req.Denom))
 		baseDenom := lock.Coins.GetDenomByIndex(0)
 
-		equivalentAmount, err := q.Keeper.GetSuperfluidOSMOTokensExcludeNative(ctx, baseDenom, lockedCoins.Amount)
+		equivalentAmount, err := q.Keeper.GetSuperfluidOSMOTokens(ctx, baseDenom, lockedCoins.Amount)
```

Comment:
Wont this include uosmo then? When I think should be excluded.

Reply:
the idea is to include everything there. I added a separate field for the non-osmo ones. To get the old value, we can subtract them. 

I thought of keeping this one as the one with only osmo and adding the field for non-osmo, but I think the query response names would become confusing: why would the field `total_equivalent_staked_amount` exclude some assets? I think "total" implies all.

I choose to change the value instead of the name here, but happy to change the api to make it more clear

Reply:
Oh I see, I was misunderstanding, thanks

Reply:
The "exclude native" really thows me off then, why it was ever named this

Reply:
I'd be happy to rename them to something with `OsmoEquivalent` to make it clearer. Just didn't want to change the existing name

Reply:
The context in which the method is being called is what is confusing me.


All things equal, in SF if I see "ExcludeNative" in context of superfluid and this new feature, I think, lets get all the superfluid staked assets of a certain validator, but leave out any osmo or token factory tokens (in this case alloyed assets). 


I think what is missing is a proper godoc here https://github.com/osmosis-labs/osmosis/blob/ae8bf49af690e9dbbbb7f090485bd82a9831d3d8/x/superfluid/keeper/twap_price.go#L59-L76 because the name could mean a variety of things

Reply:
I fully onboarded and IMO, this clarifies the difference between the two methods [abd9f3b](https://github.com/osmosis-labs/osmosis/pull/8320/commits/abd9f3b029aa40431e9431fb5bc60df9cd0e8b01)